bootstrap/src/main/java/io/airlift/bootstrap/LifeCycleStopException.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2010 Proofpoint, Inc.
```

Comment:
?
If i read `src/license/LICENSE-HEADER.txt` correctly, this line is not mandatory.

(same in other new files)

Reply:
Yes, we shouldn't add this copyright line to new files, since it isn't correct

Reply:
Sure, will remove the header from new files.